packages/react-devtools-shared/src/backend/views/TraceUpdates/canvas.js

```diff
@@ -204,6 +216,7 @@ export function destroy(agent: Agent): void {
 
 function initialize(): void {
   canvas = window.document.createElement('canvas');
+  canvas.setAttribute('popover', 'auto');
```

Comment:
Can this popover be dismissed automatically by the user? By hitting `Esc`, for example?

Reply:
Hmm, you're right. I missed that. I've switched it to manual mode and handled the elements that might interfere with the user. Thank you!